src/components/overlayCookie/overlayCookie.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
```

Comment:
Rename the file name to OverlayCookie as per the standards followed.

Reply:
PascalCase. Would be nice to have documentation too for naming. like:
`properties`: camelCase
`components`: PascalCase
`classes`: PascalCase
..
..
..